Libplanet/Blockchain/Renderers/DelayedRenderer.cs

```diff
@@ -0,0 +1,293 @@
+#nullable enable
+using System;
+using System.Collections.Concurrent;
+using System.Security.Cryptography;
+using Libplanet.Action;
+using Libplanet.Blocks;
+using Libplanet.Store;
+using Serilog;
+
+namespace Libplanet.Blockchain.Renderers
+{
+    /// <summary>
+    /// Decorates an <see cref="IRenderer{T}"/> instance and delays the events until blocks
+    /// are <em>confirmed</em> the certain number of blocks.  When blocks are recognized
+    /// the delayed events relevant to these blocks are relayed to the decorated
+    /// <see cref="IRenderer{T}"/>.
+    /// </summary>
+    /// <typeparam name="T">An <see cref="IAction"/> type.  It should match to
+    /// <see cref="BlockChain{T}"/>'s type parameter.</typeparam>
+    /// <example>
+    /// <code><![CDATA[
+    /// IStore store = GetStore();
+    /// IRenderer<ExampleAction> renderer = new SomeRenderer();
+    /// // Wraps the renderer with DelayedRenderer; the SomeRenderer instance becomes to receive
+    /// // event messages only after the relevent blocks are confirmed by 3+ blocks.
```

Comment:
I guess `confirmations` are exsist for confirmed block count. but I can't found statement check for `confirmations` only 3 more count. Is it just example or intent?

Reply:
It's example code in its XML comment.  The right next line configures `confirmations: 3`.